tests/sles4sap/publiccloud/qesap_ansible.pm

```diff
@@ -54,6 +54,19 @@ sub run {
             '/tmp/results/',
             '-type', 'f',
             '-iname', "*.xml");
+        my $find_ansible_know_issue = qr/\[OSADO\]\[softfail\] bsc#(\d+) (.*)/;
+        my $cat_command = "cat '$ret[1]'";
+        my $ansibleOutput = script_output($cat_command);
+        my ($bug_id, $description) = ("", "");
+
+        foreach my $ansible_line (split /\n/, $ansibleOutput) {
+            chomp $ansible_line;
```

Comment:
What's this `chomp` used for here?

Reply:
Sure, `chomp` will get rid of the line separation characters at the end of the line, but it's not doing anything in the context of the rest of the code. The next thing you're doing is a regexp which looks for things within the line, so the fact that you're `chomp`ing line separators right before, doesn't really help.